src/UKHO.S100PermitService.API/Controllers/PermitController.cs

```diff
@@ -25,7 +24,6 @@ public PermitController(IHttpContextAccessor httpContextAccessor,
 
         [HttpGet]
         [Route("/permits/{licenceId}")]
-        [Authorize(Policy = PermitServiceConstants.PermitServicePolicy)]
```

Comment:
Authorization of controller endpoint should be there.

Reply:
Added